src/nxt_h1proto.c

```diff
@@ -884,6 +882,34 @@ nxt_h1p_request_body_read(nxt_task_t *task, nxt_http_request_t *r)
         break;
     }
 
+    if (r->chunked) {
+        h1p->chunked_parse.mem_pool = r->mem_pool;
+        in = h1p->conn->read;
+        size = nxt_buf_mem_used_size(&in->mem);
+        if (size == 0) {
+            status = NXT_HTTP_NOT_IMPLEMENTED;
+            goto error;
```

Comment:
I notice in `nxt_h1p_peer_header_read_done` that the case where `nxt_buf_mem_used_size` returns non-zero goes to the following logic:
```c
        r->state->ready_handler(task, r, peer);
        return;
```

Does that mean we should goto ready instead?